apps/dcos_overlay/src/dcos_overlay_lashup_kv_listener.erl

```diff
@@ -136,3 +156,31 @@ to_str(Mac) when length(Mac) =:= 6 ->
     lists:flatten(string:join(List, ":"));
 to_str(IP) ->
     inet:ntoa(IP).
+
+%%%===================================================================
+%%% Metrics functions
+%%%===================================================================
+
+-spec(init_metrics() -> ok).
+init_metrics() ->
+    prometheus_counter:declare([
+        {registry, overlay},
+        {name, lashup_events_received_total},
```

Comment:
Let's not use `lashup` in metric names and help strings when it's possible

```suggestion
        {name, configuration_updates_total},
```

or maybe even better to add a summary metric for how much time it takes to process an event. it generate two time series, `num` (number of events) and `sum`. See `*_duration_seconds` in l4lb.